src/main/java/bio/terra/stairway/FlightDao.java

```diff
@@ -637,13 +657,23 @@ private void fillFlightContexts(Connection connection, List<FlightContext> fligh
                       exceptionSerializer.deserialize(rsflight.getString("serialized_exception")));
             }
 
-            flightContext.getWorkingMap().fromJson(rsflight.getString("working_parameters"));
-
             flightContext.setRerun(rsflight.getBoolean("rerun"));
             flightContext.setDirection(Direction.valueOf(rsflight.getString("direction")));
             flightContext.setResult(stepResult);
             flightContext.setFlightStatus(FlightStatus.valueOf(rsflight.getString("status")));
             flightContext.setStepIndex(rsflight.getInt("step_index"));
+
+            // TODO(PF-703): In the current transition away from working_parameters column towards
+            //               flightworking table we can either have only JSON or both.  Until we've
+            //               transitioned, delegate the decision of which to use to the FlightMap
+            //               class.
```

Comment:
whitespace nit: I don't think we're treating TODOs as indenting comments in any other repo. nit for consistency, ditto others.